pyccel/codegen/printing/ccode.py

```diff
@@ -268,6 +395,90 @@ def function_signature(self, expr):
             arg_code = ', '.join('{0}{1}'.format(self.get_declare_type(i), i) for i in expr.arguments)
         return '{0}{1}({2})'.format(ret_type, name, arg_code)
 
+    def _print_NumpyUfuncBase(self, expr):
+        """ convert a python expresion with a numpy function call to c
+        function call
+
+        Parameters:
+        -----------
+            expr (Pyccel ast node): python expresion with a numpy function call
+
+        Returns:
+        --------
+            string: equivalent of the expression in c language
+
+        Example:
+        -------
+            numpy.cos(x) ==> cos(x)
```

Comment:
This docstring is not properly formatted (see the numpydoc documentation [here](https://numpydoc.readthedocs.io/en/latest/format.html#sections)). It should read:
```
Parameters
----------
variable_name : variable_type
    Description
```
Moreover you have a typo: "expression" has two "s".
And you should capitalize letters where it's natural to do so: beginning of a sentence, "Python, "C", "Numpy".

Reply:
done.